### PR TITLE
ISSUE-598: Fix SR to run on Java 11

### PR DESCRIPTION
**Pull request by kamalcph**
* Used G1GC instead of parallel GC.
* Included JAXB-API and it's runtime jars to run the JDK8 compiled build on JRE11.

### CODE REVIEW

**Comment by kamalcph** [comment]
This PR is built on top of #597

**Comment by raju-saravanan** [comment]
@kamalcph  Can you rebase this PR with latest MASTER ?

**Comment by kamalcph** [comment]
@raju-saravanan 
Rebased the PR.

File: storage/tool/src/main/java/com/hortonworks/registries/storage/tool/sql/Utils.java [hunk 1]
```diff
@@ -32,16 +33,27 @@
...
+            final URL url = jarFile.toURI().toURL();
+            customClassLoader.addUrl(url);
+            return customClassLoader;
+        } catch (final Exception e) {
             System.out.println("Failed to load " + jarFile + " into classpath.");
             throw new Exception(e);
         }
     }
+
+    static class MyURLClassLoader extends URLClassLoader {
```
Comment (raju-saravanan) [inline storage/tool/src/main/java/com/hortonworks/registries/storage/tool/sql/Utils.java]: May be we can call this JarClassLoader or some other better name ?

File: storage/tool/src/main/java/com/hortonworks/registries/storage/tool/sql/MySqlDriverHelper.java [hunk 2]
```diff
@@ -36,10 +36,10 @@
 public class MySqlDriverHelper {
     public static final String MYSQL_JAR_FILE_PATTERN = "mysql-connector-java.*?.jar";
 
-    public static void downloadMySQLJarIfNeeded(StorageProviderConfiguration storageProperties,
-                                                String bootstrapDirPath,
-                                                String mysqlJarUrl,
-                                                Proxy proxy) throws Exception {
+    public static ClassLoader downloadMySQLJarIfNeeded(StorageProviderConfiguration storageProperties,
```
Comment (raju-saravanan) [inline storage/tool/src/main/java/com/hortonworks/registries/storage/tool/sql/MySqlDriverHelper.java]: May be we can call this loadMySQLJarIfNeeded() ?

File: bin/registry-server-start.sh [hunk 3]
```diff
@@ -73,9 +73,21 @@ else
   JAVA="$JAVA_HOME/bin/java"
 fi
 
+# GC options
+GC_LOG_FILE_NAME='registry-gc.log'
+if [ -z "$REGISTRY_GC_LOG_OPTS" ]; then
+  JAVA_MAJOR_VERSION=$($JAVA -version 2>&1 | sed -E -n 's/.* version "([0-9]*).*$/\1/p')
```
[the remark this answers is not in the document]
Comment (kamalcph) [reply bin/registry-server-start.sh]: Both of them works!

File: storage/tool/src/main/java/com/hortonworks/registries/storage/tool/sql/MySqlDriverHelper.java (hunk 2 above)
Comment (kamalcph) [reply storage/tool/src/main/java/com/hortonworks/registries/storage/tool/sql/MySqlDriverHelper.java]: This is a public method, Is it ok to change the method name?

File: storage/tool/src/main/java/com/hortonworks/registries/storage/tool/sql/Utils.java (hunk 1 above)
Comment (kamalcph) [reply storage/tool/src/main/java/com/hortonworks/registries/storage/tool/sql/Utils.java]: Renamed to JarClassLoader.

File: storage/tool/src/main/java/com/hortonworks/registries/storage/tool/sql/MySqlDriverHelper.java (hunk 2 above)
Comment (kamalcph) [reply storage/tool/src/main/java/com/hortonworks/registries/storage/tool/sql/MySqlDriverHelper.java]: Renamed to `maybeLoadMySQLJar`.

File: bin/registry-server-start.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (raju-saravanan) [reply bin/registry-server-start.sh]: @kamalcph It's a nit :)

**Pull request merged**